src/system/permissions-users-all.md

```diff
@@ -49,26 +49,39 @@ _Example Admin Users_
 
 1. Click the calendar icon to set the **Expiration Date** for the user account.
 
-    Defining an expiration date is helpful when a user or role is temporary. After the expiration date, the user account status changes to `Inactive` and can be updated, if needed.
+   Defining an expiration date is helpful when a user or role is temporary. After the expiration date, the user account status changes to `Inactive` and can be updated, if needed.
 
 1. Under _Current User Identity Verification_, enter your user account password.
 
-    {:.bs-callout .bs-callout-warning}
-    With the _Account Information_ section complete, you can save the user. The new user will show in the _Users_ grid, but the user name cannot log in until a role is assigned.
+   {:.bs-callout .bs-callout-warning}
+   With the _Account Information_ section complete, you can save the user. The new user will show in the _Users_ grid, but the user name cannot log in until a role is assigned.
 
 ## Assign a user role
 
 1. In the left panel, click **User Role**.
 
    The grid lists all the existing user roles. For a new store, _Administrators_ is the only role available.
 
+   ![Magento Admin - add new user role]({% link images/images/permissions-user-roles.png %}){: .zoom}
+   _Add New User Role_
+
 1. In the _Assigned_ column, select a user role.
 
-    You can [view existing or define additional user roles]({% link system/permissions-user-roles.md %}). After a role is defined, you must edit the user account to assign the new role.
+   You can [view existing or define additional user roles]({% link system/permissions-user-roles.md %}). After a role is defined, you must edit the user account to assign the new role.
+
+## Verify or reset 2FA provider(S)
+
+1. In the left panel, click **2FA**.
+
+   ![Magento Admin - add new user role]({% link images/images/permissions-user-2fa.png %}){: .zoom}
+   _2FA_
+
+1. Verify the 2FA solution(s) that are required for the user to sign in. Then, advise the user to install the required solution(s) before signing in to the Admin.
```

Comment:
```suggestion
1. Verify the 2FA solution(s) that are required for the user to sign in. Then, advise the user to install the required solution(s) before signing in to the _Admin_.
```

Reply:
This is actually incorrect, because multiple solutions might be available, but only one is required. I'll fix it.